build.xml

```diff
@@ -181,6 +181,23 @@
       </fileset>
     </copy>
   </target>
+	
+	
+	 <target name="jar" depends="compile.module.pearplanner">
+	    <manifestclasspath property="lib.list" jarfile="${ant.project.name}.jar">
+	        <classpath refid="external.classpath"/>
+	    </manifestclasspath>
+	    <jar destfile="${ant.project.name}.jar" basedir="${pearplanner.output.dir}">
```

Comment:
The location of the jar file is important and placing it in the project root directory makes it more likely that the file will improperly be included in a Git commit (as happened with your commit).  The `out` directory is already listed in `.gitignore` and is a good place to drop the jar.  I would add a property called `pearplanner.jaroutput.dir` that refers to a subdirectory under `out` near where `pearplanner.output.dir` is defined and then use that as the location of the jar.  Also make sure that the "clean" target handles removal.